### PR TITLE
tox: run pytest

### PR DESCRIPTION
**Pull request by goneri**
Add a pytest environment and run it against the role directory. The goal
is to add unit-test to `ansible-test-splitter` and this is the first step.

### CODE REVIEW

File: zuul.d/project.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@
       - ansible-python-jobs
     check:
       jobs:
+        - ansible-tox-py36
```
Comment (pabelanger) [inline zuul.d/project.yaml:7]: lets flip to ansible-tox-py38 given this runs under python38

File: tox.ini [hunk 2]
```diff
@@ -19,3 +19,11 @@ commands =
 setenv =
   ANSIBLE_CALLBACK_PLUGINS = {envsitepackagesdir}/ara/plugins/callbacks
 commands = {posargs}
+
+[testenv:pytest]
+deps =
+  ansible
```
Comment (pabelanger) [inline tox.ini:25]: nit, maybe change this to ansible-core, to minimize the install. But unsure what deps really look like

Comment (goneri) [reply tox.ini]: Oh +1, I always forget about that.

File: zuul.d/project.yaml [hunk 3]
```diff
@@ -4,6 +4,7 @@
       - ansible-python-jobs
     check:
       jobs:
+        - ansible-tox-py38
```
Comment (pabelanger) [inline zuul.d/project.yaml:7]: this ran tox -epy38 not tox -epytest. You'll need to set tox_envlist here

**Review by pabelanger — CHANGES_REQUESTED** [review]

**Pull request merged**